.dockerignore

```diff
@@ -0,0 +1,20 @@
+__pycache__
+*.pyc
+*.pyo
+*.pyd
+.Python
+env
+pip-log.txt
+pip-delete-this-directory.txt
+.tox
+.coverage
+.coverage.*
+.cache
+nosetests.xml
+coverage.xml
+*.cover
+*.log
+.git*
+.mypy_cache
+.pytest_cache
+.hypothesis
```

Comment:
Add all the files and folders that are not needed for the web ui app, like Dockerfile, .dockerignore, etc.